modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -1578,6 +1582,10 @@ cv::gimpl::ov::GOVExecutable::GOVExecutable(const ade::Graph &g,
     }
 }
 
+void cv::gimpl::ov::GOVExecutable::setWorkLoadType(const std::string &type)
```

Comment:
As I see in test it should be enum and not a string for OpenVINO:
https://github.com/dvpv/openvino/blob/master/src/plugins/intel_npu/tests/functional/internal/overload/compile_and_infer.hpp#L96

Please see the definition here:
https://github.com/dvpv/openvino/blob/master/src/inference/include/openvino/runtime/properties.hpp#L747
```cpp
static constexpr Property<WorkloadType, PropertyMutability::RW> workload_type{"WORKLOAD_TYPE"};
```